src/renderer/aboutDialog/js/index.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Initializes the restart dialog
+ *
+ * Creates the restart dialog once the document has been loaded.
+ * Copyright 2017 Raising the Floor - International
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ * The research leading to these results has received funding from the European Union's
+ * Seventh Framework Programme (FP7/2007-2013) under grant agreement no. 289016.
+ * You may obtain a copy of the License at
+ * https://github.com/GPII/universal/blob/master/LICENSE.txt
+ */
+
+/* global fluid */
+
+"use strict";
+(function (fluid) {
+    var gpii = fluid.registerNamespace("gpii");
+
+    var electron = require("electron");
+    var windowInitialParams = electron.remote.getCurrentWindow().params;
+
+    /**
+     * Wrapper that enables translations for the `gpii.psp.aboutDialog` component and
+     * applies interception of all anchor tags on the page, so that an external browser is used
+     * for loading them.
+     */
+    fluid.defaults("gpii.psp.translatedAboutDialog", {
+        gradeNames: ["gpii.psp.messageBundles", "fluid.viewComponent", "gpii.psp.linksInterceptor"],
+
+        components: {
+            aboutDialog: {
+                type: "gpii.psp.aboutDialog",
+                container: "{translatedAboutDialog}.container",
+                options: {
+                    model: {
+                        version:       "{translatedAboutDialog}.model.version",
+                        userListeners: "{translatedAboutDialog}.model.userListeners"
+                    },
+                    components: {
+                        // Simple placeholder required by the messageBundles
+                        channel: {
+                            type: "fluid.component"
+                        }
+                    }
+                }
+            }
+        }
+    });
+
+
+    jQuery(function () {
```

Comment:
It is a good idea to separate the definition of the `gpii.psp.translatedAboutDialog` and its instantiation in different files. That way the component will be more easily testable.